deployment/automate-testing/test.sh

```diff
@@ -0,0 +1,23 @@
+mysql -ukala -pkala < delete.sql
+rm date.txt
+cd ../..
+go run main.go serve --jobdb=mariadb --jobdb-address=\(localhost:3306\)/kala --jobdb-username=kala --jobdb-password=kala &
```

Comment:
This run with real mariadb (but works soon without real db) only to test more real... (but the problem with recurrent sure its in another place)

Reply:
Okay, sounds good?

This may or may not help: https://github.com/dolthub/go-mysql-server